package.json

```diff
@@ -10,8 +10,25 @@
     "server": "node index.js"
```

Comment:
**nitpick:** You are missing some `dev` script, so it compiles your TS files into JS on the go. Check out this: https://github.com/wclr/ts-node-dev